src/platform/site-wide/user-nav/containers/AutoSSO.jsx

```diff
@@ -1,25 +1,24 @@
 import { connect } from 'react-redux';
-import URLSearchParams from 'url-search-params';
 
-import { isLoggedIn } from 'platform/user/selectors';
 import { checkKeepAlive } from 'platform/user/authentication/actions';
 import {
   ssoeInbound,
   hasCheckedKeepAlive,
 } from 'platform/user/authentication/selectors';
+import { hasSession } from 'platform/user/profile/utilities';
 import { checkAutoSession } from 'platform/utilities/sso';
 import { removeLoginAttempted } from 'platform/utilities/sso/loginAttempted';
 
 function AutoSSO(props) {
   const {
     useInboundSSOe,
+    authenticatedWithSSOe,
```

Comment:
@hinzed1127 should `authenticatedWithSSOe` be removed?

Reply:
good catch